tests/testthat/test-post_summ.R

```diff
@@ -10,6 +10,6 @@ test_that(
     results <- post_summ(data) |> expect_no_error()
       
     # Test to ensure output is a list object:
```

Comment:
```suggestion
    # Snapshot test to ensure results are consistent:
```
(comment was from previous version of test)